cluster/manifests/zmon-agent/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "zmon-agent"
+  namespace: "kube-system"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        application: "zmon-agent"
+        version: "v0.1"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
+    spec:
+      containers:
+        - name: zmon-agent
+          image: "registry.opensource.zalan.do/zmon/zmon-agent-core:0.1-a15"
+          env:
+            - name: ZMON_AGENT_INFRASTRUCTURE_ACCOUNT
+              value: "{{ .InfrastructureAccount }}"
+            - name: ZMON_AGENT_ACCOUNT_ALIAS
+              value: "{{ .ConfigItems.account_alias }}"
```

Comment:
let's discuss if we really need this, as far as I can see it's only used for the `local` entity created by zmon-agent-core